### PR TITLE
En 8364/replace rounder with round handler

### PR DESCRIPTION
**Pull request by stefanandy**
- replace Rounder with RoundHandler
- replace rounder with roundHandler

### CODE REVIEW

File: epochStart/errors.go [hunk 1]
```diff
@@ -113,8 +113,8 @@ var ErrNilTotalDevFeesInEpoch = errors.New("total developer fees in epoch is nil
 // ErrEndOfEpochEconomicsDataDoesNotMatch signals that end of epoch data does not match
 var ErrEndOfEpochEconomicsDataDoesNotMatch = errors.New("end of epoch economics data does not match")
 
-// ErrNilRounder signals that an operation has been attempted to or with a nil Rounder implementation
-var ErrNilRounder = errors.New("nil Rounder")
+// ErrNilRoundHandler signals that an operation has been attempted to or with a nil RoundHandler implementation
+var ErrNilRoundHandler = errors.New("nil RoundHandlerField")
```
Comment (raduchis) [inline epochStart/errors.go:117]: could remove "Field
"

Comment (raduchis) [inline epochStart/errors.go:117]: could remove "Field"

File: integrationTests/mock/processComponentsStub.go [hunk 2]
```diff
@@ -17,7 +17,7 @@ type ProcessComponentsStub struct {
 	ShardCoord                     sharding.Coordinator
 	IntContainer                   process.InterceptorsContainer
 	ResFinder                      dataRetriever.ResolversFinder
-	RoundHandler                   consensus.Rounder
+	RoundHandlerField              consensus.RoundHandler
```
Comment (raduchis) [inline integrationTests/mock/processComponentsStub.go:20]: Was the "Field" really necessary here?

File: process/errors.go [hunk 3]
```diff
@@ -146,8 +146,8 @@ var ErrNilNodesConfigProvider = errors.New("nil nodes config provider")
 // ErrNilSystemSCConfig signals that nil system sc config was provided
 var ErrNilSystemSCConfig = errors.New("nil system sc config")
 
-// ErrNilRounder signals that an operation has been attempted to or with a nil Rounder implementation
-var ErrNilRounder = errors.New("nil Rounder")
+// ErrNilRoundHandler signals that an operation has been attempted to or with a nil RoundHandler implementation
+var ErrNilRoundHandler = errors.New("nil RoundHandlerField")
```
Comment (raduchis) [inline process/errors.go:150]: I think "Field" is not needed

**Review by raduchis — APPROVED** [review]

File: integrationTests/mock/processComponentsStub.go (hunk 2 above)
Comment (raduchis) [reply integrationTests/mock/processComponentsStub.go:20]: yes... there's a method with the same name

File: factory/mock/coreComponentsMock.go [hunk 4]
```diff
@@ -163,12 +163,12 @@ func (ccm *CoreComponentsMock) MinTransactionVersion() uint32 {
...
 // TxVersionChecker -
 func (ccm *CoreComponentsMock) TxVersionChecker() process.TxVersionCheckerHandler {
-		return ccm.TxVersionCheckHandler
+	return ccm.TxVersionCheckHandler
 }
 
-// Rounder -
-func (ccm *CoreComponentsMock) Rounder() consensus.Rounder {
-	return ccm.RoundHandler
+// RoundHandlerField -
```
Comment (iulianpascalau) [inline factory/mock/coreComponentsMock.go:169]: RoundHandler

File: factory/mock/processComponentsStub.go [hunk 5]
```diff
@@ -78,9 +78,9 @@ func (pcm *ProcessComponentsMock) ResolversFinder() dataRetriever.ResolversFinde
 	return pcm.ResFinder
 }
 
-// Rounder -
-func (pcm *ProcessComponentsMock) Rounder() consensus.Rounder {
-	return pcm.RoundHandler
+// RoundHandlerField -
```
Comment (iulianpascalau) [inline factory/mock/processComponentsStub.go:81]: RoundHandler

File: integrationTests/mock/processComponentsStub.go [hunk 6]
```diff
@@ -78,9 +78,9 @@ func (pcs *ProcessComponentsStub) ResolversFinder() dataRetriever.ResolversFinde
 	return pcs.ResFinder
 }
 
-// Rounder -
-func (pcs *ProcessComponentsStub) Rounder() consensus.Rounder {
-	return pcs.RoundHandler
+// RoundHandlerField -
```
Comment (iulianpascalau) [inline integrationTests/mock/processComponentsStub.go:81]: RoundHandler

File: node/mock/coreComponentsStub.go [hunk 7]
```diff
@@ -93,9 +93,9 @@ func (ccm *CoreComponentsMock) SyncTimer() ntp.SyncTimer {
 	return ccm.NtpTimer
 }
 
-// Rounder -
-func (ccm *CoreComponentsMock) Rounder() consensus.Rounder {
-	return ccm.RoundHandler
+// RoundHandlerField -
```
Comment (iulianpascalau) [inline node/mock/coreComponentsStub.go:96]: RoundHandler

**Review by iulianpascalau — COMMENTED** [review]
minor stuff

**Review by raduchis — APPROVED** [review]

**Review by iulianpascalau — APPROVED** [review]

**Pull request merged**